types/sequencify/sequencify-tests.ts

```diff
@@ -0,0 +1,34 @@
+/* Add tests for your definition file here */
+
+/// <reference types="node" />
+
+import * as sequencify from 'sequencify';
```

Comment:
Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Reply:
Even though it is not ES6 compliant, it is however working and compiling in typescript. As mentioned in the answer, it is so 2013, but the module is done that way. I cannot change the definition. I can update the code to `import sequencify = require('sequencify');`, but either way is working.

Reply:
Please use the `require` form then